extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -159,18 +159,19 @@ function () use ($limit) {
 				$db = wfGetDB( DB_SLAVE );
 
 				$moderatorIds = $this->wikiService->getWikiModeratorIds( 0, false, true, null );
+				if ( empty ( $moderatorIds ) ) {
+					return [];
+				}
+
 				$adminIds = $this->wikiService->getWikiAdminIds( 0, false, true, null, true );
 				$botIds = $this->getBotIds();
 				$dateTwoYearsAgo = date( 'Y-m-d', strtotime( '-2 years' ) );
 
 				$sqlData = ( new WikiaSQL() )
 					->SELECT( 'distinct rev_user' )
 					->FROM( 'revision' )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "revision" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)